packages/widget/src/components/TokenList/TokenListItem.tsx

```diff
@@ -101,7 +103,12 @@ export const TokenListItemButton: React.FC<TokenListItemButtonProps> = ({
       dense
     >
       <ListItemAvatar>
-        <Avatar src={token.logoURI} alt={token.symbol}>
+        <Avatar
+          src={token.logoURI}
+          alt={token.symbol}
+          sx={!imageLoaded ? { bgcolor: grey[400] } : null}
```

Comment:
For consistency, it would be nice to use theme.vars with grey.300 here so it can be customized. We use grey.300 in other places for avatar placeholders, and we don't really use grey.400 for anything.